src/main/java/com/github/jchartier/slackbot/pomodoro/command/PomodoroCommand.java

```diff
@@ -0,0 +1,8 @@
+package com.github.jchartier.slackbot.pomodoro.command;
+
+import com.ullink.slack.simpleslackapi.SlackUser;
+
+public interface PomodoroCommand {
+
```

Comment:
We should add the help message in every Pomodoro commands. It will help to:

- Build the `help` command
- Give some feedback when the command fails